meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/apalis-imx8/lmp-base.cfg

```diff
@@ -18,6 +18,9 @@ CONFIG_SPL_DRIVERS_MISC=y
 CONFIG_SPL_SERIAL=y
 CONFIG_RSA=y
 CONFIG_HASH=y
+CONFIG_TEE=y
+CONFIG_OPTEE=y
```

Comment:
Apalis now lacks optee node, so OPTEE won't be proper.
I'll create PR

arch/arm/dts/fsl-imx8qm-mek.dts

Reply:
https://github.com/foundriesio/u-boot/pull/191